When creating a solr cluster passing in a cluster size fails.

When creating a solr cluster passing in a cluster size fails.  It may be a case where we just need to import long
```
retrieve_and_rank.create_solr_cluster(cluster_name='pythonCluster', cluster_size=10)
```
```
self = <watson_developer_cloud.retrieve_and_rank_v1.RetrieveAndRankV1 object at 0x0000000003BC1940>
cluster_name = 'pythonCluster', cluster_size = 10

    def create_solr_cluster(self, cluster_name=None, cluster_size=None):
>       if cluster_size and isinstance(cluster_size, (int, long)):
E       NameError: name 'long' is not defined

File "C:\Users\IBM_ADMIN\Documents\Work\Watson\DevExp\python-sdk\watson_developer_cloud\retrieve_and_rank_v1.py", line 36
NameError    
```

--- a/watson_developer_cloud/retrieve_and_rank_v1.py
+++ b/watson_developer_cloud/retrieve_and_rank_v1.py
@@ -32,7 +32,7 @@ class RetrieveAndRankV1(WatsonDeveloperCloudService):
         return self.request(method='GET', url='/v1/solr_clusters', accept_json=True)
 
     def create_solr_cluster(self, cluster_name=None, cluster_size=None):
-        if cluster_size and isinstance(cluster_size, (int, long)):
+        if cluster_size:
             cluster_size = str(cluster_size)
         params = {'cluster_name': cluster_name, 'cluster_size': cluster_size}
         return self.request(method='POST', url='/v1/solr_clusters', accept_json=True, json=params)

